generators/dart.js

```diff
@@ -71,11 +71,18 @@ Blockly.Dart.ORDER_CASCADE = 15;       // ..
 Blockly.Dart.ORDER_ASSIGNMENT = 16;    // = *= /= ~/= %= += -= <<= >>= &= ^= |=
 Blockly.Dart.ORDER_NONE = 99;          // (...)
 
+/**
+ * Whether the init method has been called.
+ * @type {?boolean}
+ */
+Blockly.Dart.isInitialized = false;
+
 /**
  * Initialise the database of variable names.
  * @param {!Blockly.Workspace} workspace Workspace to generate code from.
  */
 Blockly.Dart.init = function(workspace) {
+  this.isInitialized = true;
```

Comment:
Just want to call out that I'm not sure whether this ought to be at the end of the init function, rather than the start.

Reply:
Done.